include/TRK/serpoll.h

```diff
@@ -1,6 +1,7 @@
 #ifndef TRK_SERPOLL
 #define TRK_SERPOLL
 #include "types.h"
+#include "TRK/trk.h"
```

Comment:
One of the cases I found where if the `.h` file wasn't declared when context was generated the dependencies wouldn't be expanded in the right order.